docs/platforms/apple/common/features/index.mdx

```diff
@@ -1,3 +1,18 @@
+---
+title: Features
+sidebar_order: 1
+description: "Learn about the features of Sentry's Apple SDK."
+---
+
+Sentry's Apple SDK enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The below, is a list of features that are available as part of this SDK.
+
+<Note>
+
+The SDK builds a crash report that persists to disk and tries to send the report right after the crash. Since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+</Note>
```

Comment:
```suggestion
<Note>

The SDK builds a crash report that persists to disk. While it attempts to send the report right after the crash, it may not always work because the environment may be unstable at the time of the crash. If this is the case, the report will be sent upon application start. 

</Note>
```